Portfolio: finish mobile version

khanh-devos opened the pull request:
This will have 2 parts: Works and About Me
- Works has a work box that is repeated 6 times. 
- About Me has 3 main parts: resume, skills, contacts
- A button type is used many times

khanh-devos committed:
Portfolio: Finish Mobile Version

- Set Layout for Works section in <main>
- Padding 120px 24px for Works, 12px is general margin for each items inside
- Setup 2 classes, nav-buttons and btn-type-1 for furture usage.
- Finishing class "work-box" that repeat many times in Works: fonts, euse nav-buttons.

khanh-devos committed:
Merge branch 'master' into build-Works-and-About-Me

- fix conflicts at <link> for fonts
- update new change of README file
- Update new change of part "headers" and "headline"

khanh-devos committed:
Build part "Language Framework Skills"

- use mainly ul as a parent, div and article

khanh-devos committed:
Build contact section

- form with input and textarea
- reuse button-type-1
- reuse social media

khanh-devos committed:
fix stylelint error

Kamba56 commented on index.html:
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
+
     </section>
 
+    <section id="about-me">
```
- [x] There is a background in your about section that is not on the Figma template. Kindly remove this, please. You are expected to replicate the figma template as much as possible, this includes background images.

- [x] As demanded by the project requirements, kindly use flexbox to layout contents of the about section.
### Figma
![image](https://user-images.githubusercontent.com/94551484/235303444-da3f7e2a-0c0a-43a4-976f-89c0f20332e1.png)
### Your design
![image](https://user-images.githubusercontent.com/94551484/235303490-edf06f64-5ad3-4d1c-ad57-3d87600a5fda.png)

Kamba56 commented on index.html:
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
+
+      </div>
+
+    </section>
```
- [x] A milestone for adding the contact form is coming in the following week. Kindly remove this for now and add it in the coming milestone. You are expected to build your portfolio one milestone at a time.

Kamba56 commented on index.html:
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
+          <a href="">API Design</a>
+          <a href="">Web Development</a>
+        </div>
+      </ul>
```
- [ ] Great job using an unordered list to list the categories of technologies. Kindly use unordered list to list the technologies under each category. For example, for the Language category;
```html
<ul>
  <li>Languges
    <ul>
      <li>Javascript</li>
      <li>HTML</li>
      <li>CSS</li>
      <li>Ruby</li>
    </ul>
  </li>
....
</ul>
```

Kamba56 commented on styles/index.css:
```diff
@@ -12,7 +12,181 @@ body {
   background-color: #e5e5e5;
```
- [x] The background color of the body in the figma template is white. Kindly change this to white. You are expected to stick to design template as much as possible, this includes colors, background colors, fonts, sizes, margins and paddings.

Kamba56 commented on styles/index.css:
```diff
@@ -98,47 +262,198 @@ body {
-#headline-nav {
-  margin: 0 0;
+/* Works */
+#works { padding: 120px 24px; }
```
- [x] As demanded by the project requirements, kindly use CSS grid to layout the contents of your work section. Grid helps in creating a responsive web page so it is important you practice using it.

Kamba56 commented on styles/index.css:
```diff
@@ -12,7 +12,181 @@ body {
+}
+
+/* class works-box */
+.works-box {
```
- [x] As demanded by the project requirements, kindly use flexbox to layout contents of each work card. Flexbox makes building responsive applications easier.

Kamba56 commented on index.html:
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
 
+    <section id="about-me">
+      <div id="resume">
+        <h1>About me</h1>
```
- [x] Kindly use only one h1 tag in your web application. It is best SEO practice to use a single h1 tag per web page. Kindly change this to another heading tag `h2 - h5`.

Kamba56 commented on index.html:
```diff
@@ -34,17 +35,189 @@ <h2>I am a Software Engineer</h2>
+
+    <section id="works">
+      <div id="works-intro">
+        <h1>My Recent Works</h1>
```
- [x] Kindly use only one h1 tag in your web application. It is best SEO practice to use a single h1 tag per web page. Kindly change this to another heading tag `h2 - h5`.

Kamba56 commented on styles/index.css:
```diff
@@ -12,7 +12,181 @@ body {
+.nav-buttons li {
+  height: 40px;
+  width: auto;
+  border: 1px solid #8993a4;
```
- [ ] Kindly remove this border as it is not in the figma template. You are expected to stick to design template as much as possible.
### Your design
![image](https://user-images.githubusercontent.com/94551484/235304355-2bb3f482-d501-4eea-9c9b-a5f62d6b08de.png)
### Figma
![image](https://user-images.githubusercontent.com/94551484/235304371-b2e30eee-a366-45df-9bff-d84d5688e783.png)

Kamba56 reviewed (changes requested):
Hi @khanh-devos, 

Good job so far!!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- [x] Great job using flexbox to layout contents!👏👏👏
- [x] You did a great job replicating the template. 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

khanh-devos committed:
Fix Requirement of Code Review

- change background, use flex layout of About me section
- remove contact
- use nest list, ul in li
- change background to white
- use Grid layout for Work section
- use flex layout for work-card
- Only use h1 one 1 time due to SEO
- Remove border of box-nav at work card

khanh-devos committed:
fix Webhint error

LionRouge1 reviewed (commented):
Hi @khanh-devos, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

![done](https://media.giphy.com/media/a5llhdWKMOIfneMCpB/giphy.gif)

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
